tests/test_overscanCorrection.py

```diff
@@ -44,6 +44,9 @@ def testOverscanCorrectionY(self, **kwargs):
         maskedImage = afwImage.MaskedImageF(bbox)
         maskedImage.set(10, 0x0, 1)
 
+        dataBox = afwGeom.Box2I(afwGeom.Point2I(0, 0), afwGeom.Extent2I(10, 10))
+        dataImage = afwImage.MaskedImageF(maskedImage, dataBox)
```

Comment:
Is the point of `dataImage` just to be a small chunk of `maskedImage`? Does it save a lot of time to run the overscan correction test on this rather than the entire `maskedImage` throughout?

Reply:
`maskedImage` includes the overscan, so if I were to operate on that I would be subtracting the overscan twice, which breaks the checks.